rxbinding/src/main/java/com/jakewharton/rxbinding/widget/TextViewEditorActionEvent.java

```diff
@@ -27,23 +28,27 @@ public int actionId() {
     return actionId;
   }
 
-  @NonNull
-  public KeyEvent keyEvent() {
+  /**
+   * If triggered by an enter key, this is the event, otherwise {@code null}.
+   *
+   * @see TextView.OnEditorActionListener#onEditorAction(android.widget.TextView, int,
+   * android.view.KeyEvent)
+   */
+  @Nullable public KeyEvent keyEvent() {
     return keyEvent;
   }
 
   @Override public boolean equals(Object o) {
-    if (o == this) return true;
-    if (!(o instanceof TextViewEditorActionEvent)) return false;
-    TextViewEditorActionEvent other = (TextViewEditorActionEvent) o;
-    return other.view() == view() && other.actionId == actionId && other.keyEvent.equals(keyEvent);
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
```

Comment:
This type is final so an `instanceof` check can be performed here instead of class comparison.